js_modules/dagster-ui/packages/ui-core/src/jobs/JobsPageContent.tsx

```diff
@@ -164,7 +164,7 @@ export const JobsPageContent = () => {
         </Box>
       ) : null}
       {loading && !repoCount ? (
-        <Box padding={64}>
+        <Box padding={64} flex={{direction: 'row', justifyContent: 'center'}}>
           <SpinnerWithText label="Loading jobs…" />
         </Box>
```

Comment:
I think maybe this has been left-aligned (incorrectly) for a while, but we haven't noticed because we're not looking at Jobs as often as we used to.